src/StoryDeck.elm

```diff
@@ -58,37 +58,18 @@ storyTeaser deckId =
         t =
             toString
     in
-    div [ class "card" ]
-        [ a
-            [ href ("#/stories/" ++ String.fromInt deckId)
-            , onClick (ButtonPress "story" "view-single" (t (storyTitle deckId)) False)
-            ]
-            [ img [ class "card--thumbnail", src (storyTeaserImgPath deckId), alt (t (storyTeaserImgAltText deckId)) ] []
-            ]
-        , a
-            [ class "link--unstyled"
-            , href ("#/stories/" ++ String.fromInt deckId)
-            , onClick (ButtonPress "story" "view-single" (t (storyTitle deckId)) False)
-            ]
-            [ h3 [ class "title--small" ] [ text (t (storyTitle deckId)) ]
-            ]
+    a
+        [ class "card link--unstyled", href ("#/stories/" ++ String.fromInt deckId), onClick (ButtonPress "story" "view-single" (t (storyTitle deckId)) False) ]
+        [ img [ class "card--thumbnail", src (storyTeaserImgPath deckId), alt (t (storyTeaserImgAltText deckId)) ] []
+        , h3 [ class "title--small" ] [ text (t (storyTitle deckId)) ]
         , blockquote [ class "card--quote" ]
             [ text (t (getDeck deckId decks).teaser) ]
         , div [ class "text-right text-with-icon--right stories--more-link" ]
-            [ a
-                [ class "link"
-                , href ("#/stories/" ++ String.fromInt deckId)
-                , onClick (ButtonPress "story" "view-single" (t (storyTitle deckId)) False)
-                ]
+            [ span
```

Comment:
`a` element `href` here.

Reply:
I think it's better not to have link within a link (or a link within a clickable div), but again @Nikomus @falconx you probably have more up to date knowledge than I in this area.